CARET_trace/src/ros_trace_points.cpp

```diff
@@ -51,6 +52,81 @@
 std::unique_ptr<std::thread> trace_node_thread;
 thread_local bool trace_filter_is_rcl_publish_recorded;
 
+using std::string;
+using std::vector;
+static bool ignore_rcl_timer_init = false;
+
+static vector<string> string_split(string & str, char delim)
+{
+  using std::stringstream;
+  vector<string> elems;
+  stringstream ss(str);
+  string item;
+  while (getline(ss, item, delim)) {
+    if (!item.empty()) {
+      elems.push_back(item);
+    }
+  }
+  return elems;
+}
+
+static bool is_ros2_launch_command()
+{
+  // Caret_trace's <TraceNode> should not record rcl_timer_init() for command line
+  // "python3" launch.
+  // This is because there are multiple processes that repeat spin_once() in a short time
+  // in the process started by "python3".
+  // If this is not excluded, the end time will increase as time passes
+  // after Caret_trace starts.
```

Comment:
I think this comment is not required because the contents of this function is staightforward unlike previous version.

```suggestion
```

Reply:
It has been deleted